### ISSUE TITLE
External property inconsistently not displaying

### ISSUE THREAD

**Issue by mrxbox98** [SpiGetOrg/Spiget#51]
In the ``/resources`` endpoint the ``external`` property representing an external download link randomly doesn't appear.
Example:
![](https://i.mrxbox98.me/file/2022/06/chrome_gYzFpZG6MR.png)
![](https://i.mrxbox98.me/file/2022/06/chrome_5PdpKsaZ5f.png)

**Comment by InventivetalentDev** [comment]
can you please check the response headers and look for `x-spiget-server`?

**Comment by mrxbox98** [comment]
> can you please check the response headers and look for `x-spiget-server`?

Yep, I tried it a view times and all the times the ``external`` property was missing the spiget server was ``avocado``
